### ISSUE TITLE
Incorrect statement

### ISSUE THREAD

**Issue by vkurchatkin** [piecioshka/test-flow-vs-typescript#3]
This statement:

> Flow was created to keep proper type during runtime

is incorrect. Flow doesn't check types at runtime or anything like that. In your test you are not using Flow, you are using `babel-plugin-tcomb` which has nothing to do with Flow.

**Comment by piecioshka** [comment]
@vkurchatkin yes I know. So for what is FlowType?

**Comment by vkurchatkin** [comment]
Flow is this: https://github.com/facebook/flow It's a static analysis tool, not a transpiler or runtime system

**Comment by samwgoldman** [comment]
@vkurchatkin is right. Flow does not insert any runtime type checks on its own, just like TypeScript.

(Also, Flow supports classes just fine, so you can use a class in the Flow example, too.)

**Comment by piecioshka** [comment]
You're right, and thanks for taking the time to explain it. This was an early experiment and the example conflated `babel-plugin-tcomb` (which does inject runtime checks) with Flow itself. Flow is a static analysis tool and doesn't add any runtime type checking, same as TypeScript — so the statement was simply wrong. Appreciate the correction from you and @samwgoldman.

**Comment by piecioshka** [comment]
Fixed in #9 — the README now states that Flow is a static analysis tool (no runtime checks), and clarifies that the runtime error in the example came from `babel-plugin-tcomb`, not Flow.
